### DISCUSSION TITLE
PDF Extract Images (OCR) Fails with 'Unsupported Filter' on Scanned Documents

### DISCUSSION THREAD

**Discussion by Genai-labs** [open-webui/open-webui#5178] [General]
**Is your feature request related to a problem? Please describe.** 

In the Admin Dashboard, under the Settings > Document section, when using the "PDF Extract Images (OCR)" option, the process fails for scanned documents and returns the error message: "Unsupported filter". This issue specifically occurs with documents that have been printed and then scanned back into PDF format.

**Describe alternatives you've considered**

 It could be beneficial to have an extraction method that supports this type of document.

**Additional context**

version : 3.18

![image](https://github.com/user-attachments/assets/2453f667-f9c3-49ef-ba4d-95fe3076b178)

**Comment by justinh-rahb** [comment]
Use Tika for content extraction instead, it's OCR is more reliable.
